### ISSUE TITLE
Cancel all orders

### ISSUE THREAD

**Issue by CryptoKong** [Codaone/DEXBot#405]
Add a "cancel all orders" button to the dash for each worker next to "modify worker" button, when playing around with workers and trying to tweak settings in the GUI it is a pain to keep having to go to the DEX and leave the exchange page to go to the open orders page to cancel all orders and start again. If this button was available on the dash for each worker it would be much better.

**Comment by bitfag** [comment]
Agree, I personally use simple script which cancels all orders on the defined account.

**Comment by thehapax** [comment]
ok i'll also look into this GUI issue. @MarkoPaasila
